data/json/recipes/armor/head.json

```
@@ -120,6 +120,32 @@
     "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "forge", 50 ], [ "oxy_torch", 10 ] ] ],
     "components": [ [ [ "gold_small", 300 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "drinking_hat",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_HEAD",
+    "skill_used": "fabrication",
+    "time": "5 m",
+    "reversible": true,
+    "autolearn": true,
+    "using": [ [ "cordage_short", 2 ] ],
+    "components": [
+      [
+        [ "can_drink_unsealed", 2 ],
+        [ "can_food_unsealed", 2 ],
+        [ "canister_empty", 2 ],
+        [ "bottle_plastic_small", 2 ],
+        [ "sippy_cup", 2 ],
+        [ "clay_canister", 2 ],
+        [ "flask_glass", 2 ],
+        [ "beaker", 2 ],
+        [ "flask_hip", 2 ]
+      ],
+      [ [ "hose", 1 ] ]
```

```suggestion
      [ [ "hose", 1 ] ],
      [ [ "wire", 2 ] ]
```

I imagine it's some kind of headband with cans strapped to a ring? Wires should help with that.

---

previously, the binding material was duct tape :)

---

I went with 1 unit of wire; judging by weight, 2 units seem to be more than would be needed.